### PR TITLE
Added changelog entry for issue #22208

### PR DESCRIPTION
**Pull request by cheweytoo**
This adds a changelog entry for PR #22239 / isse #22208.

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - Feature: [OpenMusic#54] Added Progressive ride music style (feat. Approaching Nirvana).
 - Change: [#22230] The plugin/script engine is now initialised off the main thread.
 - Change: [#22251] Hide author info in the scenery window unless debug tools are active.
+- Fix: [#22208] Area edit cursors can disappear in some cases.
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:10]: We usually call these tool selection areas. There were particular cases involved, right? Just the map edge?

Comment (cheweytoo) [reply distribution/changelog.txt:10]: > We usually call these tool selection areas.

Right. So change to "Tool selection areas can disappear in some cases."?

> There were particular cases involved, right? Just the map edge?

That's a different issue, and would be #22209 ;-)

**Comment by mrmbernardi** [comment]
The bug was actually that the cursor wouldn't register hits of any sort. It wasn't just limited to the land and other similar tools.

**Comment by cheweytoo** [comment]
> The bug was actually that the cursor wouldn't register hits of any sort.

"Fix: [#22208] Cursor may fail to register hits in some cases."?

**Comment by mrmbernardi** [comment]
I like `Cursor` instead of `Cursors`, otherwise that's good :)

**Review by mrmbernardi — APPROVED** [review]

**Comment by J-PIE-314** [comment]
This changelog entry should be updated to end with the text ` (original bug).` See: https://github.com/OpenRCT2/OpenRCT2/issues/22208#issuecomment-2227407904.

**Pull request merged**